### ISSUE TITLE
OLM install fails, CRD file not found

### ISSUE THREAD

**Issue by jkapponms** [operator-framework/operator-sdk#5993]
## Bug Report

**What did you do?**
Ran `operator-sdk olm install` to install OLM into a fresh [kind](https://kind.sigs.k8s.io/) cluster.

**What did you expect to see?**
OLM successfully installed into my cluster. 

**What did you see instead? Under which circumstances?**
```
$ operator-sdk olm install
INFO[0000] Fetching CRDs for version "latest"           
INFO[0000] Fetching resources for resolved version "latest" 
FATA[0000] Failed to install OLM version "latest": failed to get resources: failed to fetch CRDs: request failed: failed GET 'https://github.com/operator-framework/operator-lifecycle-manager/releases/latest/download/crds.yaml': unexpected status code 404, expected 200; manifests may not exist for this OLM release,please check https://github.com/operator-framework/operator-lifecycle-manager/releases for olm.yaml and crds.yaml
```

Pulled the url `https://github.com/operator-framework/operator-lifecycle-manager/releases/latest/download/crds.yaml` and popping it into a browser, and looking at OLM's recent v0.22.0 release, this file indeed doesn't exist.

**Comment by Chever-John** [comment]
Now, I think it has them.

https://github.com/operator-framework/operator-lifecycle-manager/issues/2841

**Comment by varshaprasad96** [comment]
Looks like OLM release has the required manifests now: https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/v0.22.0. @jkapponms Is the command working now?

Meanwhile, we need to update Operator SDK to officially support the latest OLM release version by updating bindata (https://sdk.operatorframework.io/docs/overview/#olm-version-compatibility)

**Comment by everettraven** [comment]
I tested the command locally and everything seems to be good. 

I think the bindata update should be a part of the release process so it should be updated whenever we cut the 1.23.0 release right @varshaprasad96 ?

**Comment by jmrodri** [comment]
In my local branch, it works after updating bindata.

```
[jesusr@transam operator-sdk{update-olm-bindata}]$ operator-sdk version
operator-sdk version: "v1.22.0-36-geb509c80", commit: "eb509c80b5de48e7c51127a8347b091822726612", kubernetes version: "v1.24.2", go version: "go1.18.5", GOOS: "linux", GOARCH: "amd64"
[jesusr@transam operator-sdk{update-olm-bindata}]$ operator-sdk olm install
INFO[0000] Fetching CRDs for version "latest"           
INFO[0000] Fetching resources for resolved version "latest" 
I0822 21:47:30.702503   10114 request.go:601] Waited for 1.046760883s due to client-side throttling, not priority and fairness, request: GET:https://127.0.0.1:36967/apis/autoscaling/v2?timeout=32s
INFO[0008] Creating CRDs and resources                  
INFO[0008]   Creating CustomResourceDefinition "catalogsources.operators.coreos.com" 
INFO[0008]   Creating CustomResourceDefinition "clusterserviceversions.operators.coreos.com" 
INFO[0008]   Creating CustomResourceDefinition "installplans.operators.coreos.com" 
INFO[0008]   Creating CustomResourceDefinition "olmconfigs.operators.coreos.com" 
INFO[0008]   Creating CustomResourceDefinition "operatorconditions.operators.coreos.com" 
INFO[0008]   Creating CustomResourceDefinition "operatorgroups.operators.coreos.com" 
INFO[0008]   Creating CustomResourceDefinition "operators.operators.coreos.com" 
INFO[0008]   Creating CustomResourceDefinition "subscriptions.operators.coreos.com" 
INFO[0008]   Creating Namespace "olm"                   
INFO[0008]   Creating Namespace "operators"             
INFO[0008]   Creating ServiceAccount "olm/olm-operator-serviceaccount" 
INFO[0008]   Creating ClusterRole "system:controller:operator-lifecycle-manager" 
INFO[0008]   Creating ClusterRoleBinding "olm-operator-binding-olm" 
INFO[0008]   Creating OLMConfig "cluster"               
INFO[0012]   Creating Deployment "olm/olm-operator"     
INFO[0012]   Creating Deployment "olm/catalog-operator" 
INFO[0012]   Creating ClusterRole "aggregate-olm-edit"  
INFO[0012]   Creating ClusterRole "aggregate-olm-view"  
INFO[0012]   Creating OperatorGroup "operators/global-operators" 
INFO[0012]   Creating OperatorGroup "olm/olm-operators" 
INFO[0012]   Creating ClusterServiceVersion "olm/packageserver" 
INFO[0012]   Creating CatalogSource "olm/operatorhubio-catalog" 
INFO[0012] Waiting for deployment/olm-operator rollout to complete 
INFO[0012]   Waiting for Deployment "olm/olm-operator" to rollout: 0 of 1 updated replicas are available 
INFO[0020]   Deployment "olm/olm-operator" successfully rolled out 
INFO[0020] Waiting for deployment/catalog-operator rollout to complete 
INFO[0020]   Deployment "olm/catalog-operator" successfully rolled out 
INFO[0020] Waiting for deployment/packageserver rollout to complete 
INFO[0020]   Waiting for Deployment "olm/packageserver" to rollout: 0 of 2 updated replicas are available 
INFO[0046]   Deployment "olm/packageserver" successfully rolled out 
INFO[0047] Successfully installed OLM version "latest"  

NAME                                            NAMESPACE    KIND                        STATUS
catalogsources.operators.coreos.com                          CustomResourceDefinition    Installed
clusterserviceversions.operators.coreos.com                  CustomResourceDefinition    Installed
installplans.operators.coreos.com                            CustomResourceDefinition    Installed
olmconfigs.operators.coreos.com                              CustomResourceDefinition    Installed
operatorconditions.operators.coreos.com                      CustomResourceDefinition    Installed
operatorgroups.operators.coreos.com                          CustomResourceDefinition    Installed
operators.operators.coreos.com                               CustomResourceDefinition    Installed
subscriptions.operators.coreos.com                           CustomResourceDefinition    Installed
olm                                                          Namespace                   Installed
operators                                                    Namespace                   Installed
olm-operator-serviceaccount                     olm          ServiceAccount              Installed
system:controller:operator-lifecycle-manager                 ClusterRole                 Installed
olm-operator-binding-olm                                     ClusterRoleBinding          Installed
cluster                                                      OLMConfig                   Installed
olm-operator                                    olm          Deployment                  Installed
catalog-operator                                olm          Deployment                  Installed
aggregate-olm-edit                                           ClusterRole                 Installed
aggregate-olm-view                                           ClusterRole                 Installed
global-operators                                operators    OperatorGroup               Installed
olm-operators                                   olm          OperatorGroup               Installed
packageserver                                   olm          ClusterServiceVersion       Installed
operatorhubio-catalog                           olm          CatalogSource               Installed
```

**Comment by everettraven** [comment]
@jmrodri @varshaprasad96 are we good to close this issue? I noticed that this was marked as a release blocker for the 1.23.0 release and since we have released 1.23.0 I assume this is no longer an issue?

**Comment by asmacdo** [comment]
I'm going to assume @everettraven is right :)
